src/CoinParamUtils.cpp

```diff
@@ -129,6 +130,129 @@ std::string nextField(const char *prompt)
 
 namespace CoinParamUtils {
 
+std::string printString(const std::string input, int maxWidth)
+{
+   std::ostringstream buffer;
+   std::istringstream iss(input);
+   int currentWidth = 0;
+   std::string word;
+   while (iss >> word){
+      currentWidth += word.length();
+      if (currentWidth >= 65){
+         buffer << std::endl << word;
+      } else {
+         buffer << word;
+      }         
+   }
+   return buffer.str();
+}
+
+std::string printString(const char *input, int maxWidth)
+{
+   return CoinParamUtils::printString(std::string(input));
+}
+
+void
+readFromStream(std::deque<std::string> &inputQueue,
+               std::istream &inputStream)
+{
+   std::string field;
+   while (inputStream >> field){
+      std::string::size_type found = field.find('=');
+      if (found != std::string::npos) {
+         inputQueue.push_back(field.substr(0, found));
+         inputQueue.push_back(field.substr(found + 1));
+      } else {
+         inputQueue.push_back(field);
+      }
+   }
+}
+
+void
+readInteractiveInput(std::deque<std::string> &inputQueue,
+                     std::string prompt)
```

Comment:
passing in a (possibly const) reference would safe a string copy. I won't comment that on the other functions in this file, I can see a pattern.

Reply:
I assume you mean for passing in `prompt`? Yes, makes sense. I wasn't super careful about const, etc. in doing this. Wanted to get something working first, then clean up.